src/styles/blocks/calendar.scss

```
@@ -0,0 +1,46 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $space-between;
+  width: week-width($side--size, $days-in-week);
+  padding: 10px;
+  margin: 0 auto;
+
+  &__day {
+    @include text-in-center;
+    @include square;
+
+    background-color: #eee;
+    border: 1px solid #000;
+
+    transition: background-color 0.5s, transform 0.5s ease-in-out;
```

```suggestion
    transition: background-color $transitions-duration, transform  $transitions-duration ease-in-out;
```
it is a good approach to move such reusable values into variables